packages/core/src/Stores/notification-store.js

```diff
@@ -52,6 +52,9 @@ export default class NotificationStore extends BaseStore {
                 root_store.modules?.cashier?.general_store?.is_p2p_visible,
                 root_store.common?.selected_contract_type,
                 root_store.client.is_eu,
+                root_store.client.is_uk,
```

Comment:
a previously used `init()` was called before the entire app state update. that's why I have to update notifications here in a reaction when any of the above values from different stores are set.